bot/src/commands/who.rs

```diff
@@ -7,31 +8,18 @@ use crate::types::{Context, Error};
 async fn who(
     ctx: Context<'_>,
     #[description = "Search for a nation or alliance"] search: String,
+    #[description = "If to search for an alliance"] is_alliance: Option<bool>,
```

Comment:
This works for now, in the future I'd like to create a union enum to facilitate a cleaner user experience.

Reply:
for sure, this command isnt finished, i just wanted to move on to somthing else :P